veracruz-mcu-client/run_markdown.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+# AUTHORS
+#
+# The Veracruz Development Team.
+#
+# COPYRIGHT
+#
+# See the `LICENSE.markdown` file in the Veracruz root directory for licensing
+# and copyright information.
+#
+
+# Runs the given markdown file as a bash script, with any failing commands
```

Comment:
Hmm, I don't think we should be inventing our own literate programming tool for Bash :).  Can you factor out the inline Bash comments in the Markdown file and add them to an explicit file under version control?

Reply:
Looks like I'm outvoted here, I can commit a copy of the markdown that is just a bash script.

I'm still of the opinion this is going to be less reliable, the motivation of this is to test the example documentation, and having separate files means they will fall out of sync over time.

Reply:
Will also need to update the run_cli_test.sh currently in main, can create a separate PR for that